configure.ac

```diff
@@ -30,7 +30,7 @@ AC_SUBST([PACKAGE_RPM_VERSION])
 PACKAGE_SHORT_VERSION="$(echo VERSION_NUMBER | cut -d '.' -f 1,2,3)"
 AC_SUBST([PACKAGE_SHORT_VERSION])
 
-PACKAGE_RPM_RELEASE="0.0.$(echo VERSION_SUFFIX | sed 's/^_//')"
+PACKAGE_RPM_RELEASE="0.1.$(echo VERSION_SUFFIX | sed 's/^_//')"
```

Comment:
Is this really needed?

Reply:
it help ensuring the update is visible but given we are building with a time stamp we can live without it.